src/BuildScriptGenerator/DotNetCore/DotNetCoreManifestFilePropertyKeys.cs

```diff
@@ -0,0 +1,12 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator.DotNetCore
+{
+    internal static class DotNetCoreManifestFilePropertyKeys
+    {
+        internal const string StartupDllFileName = nameof(StartupDllFileName);
```

Comment:
our prop key names are spread across multiple files - maybe we should (at least aim to) consolidate them?

Reply:
These property keys are specific to DotNetCore, so I put them in a separate file. I think we should do similarly for other platforms. Yes, keys which are common across all platforms (ex: OperationID) can be put in a common file. Do you agree?